package.json

```diff
@@ -165,7 +165,9 @@
     "nth-check": "^2.0.1",
     "browserslist": "^4.16.5",
     "json-schema": "^0.4.0",
-    "node-forge": "^1.0.0"
+    "node-forge": "^1.0.0",
+    "cross-fetch": "^3.1.5",
+    "node-fetch": "^2.6.7"
```

Comment:
This list keep getting longer and longer. I suppose at some point when the libraries using these deps are updated, these deps no longer have to be mentioned here. But that is a manual revision I gues.

Reply:
Yes, you are correct, we also have to keep an eye on that.